### ISSUE TITLE
Disable window manager?

### ISSUE THREAD

**Issue by borgauf** [daedreth/UncleDavesEmacs#12]
How would I disable booting into the window manager? I don't want to guess and screw something up.

**Comment by yisraeldov** [comment]
If you have another window manager running exwm won't start

**Comment by borgauf** [comment]
True, but it always asks me y or n if I want to run it. I choose n, then gives an error in a buffer. I quit the buffer and all is fine. But I'd like to avoid the whole thing if possible.

**Comment by yisraeldov** [comment]
Do just comment out the exwm stuff

**Comment by daedreth** [comment]
I bundled all the exwm stuff into one codeblock, you can remove it in its entirety to disable the window managing functionality altogether!

At some point I plan on adding a selective menu upon initial launch to enable/disable features, stay tuned for that!
